### DISCUSSION TITLE
Ubuntu on the Icicle Kit

### DISCUSSION THREAD

**Discussion by rockoguido** [polarfire-soc/polarfire-soc-documentation#295] [Linux build systems for PolarFire SoC] [answered]
Hi.. hope I'm in the right category, just trying to check on this Canonical release.    I understood the launch was for today, the 24th.    I'm guessing there is a slight delay which is fine, of course.    
Any update is appreciated, I'm keen on checking this out...
Thx!
Dave

**Comment by ConchuOD** [comment, marked as answer]
It's been released, and some documentation on how to get up and running can be found here:
https://github.com/polarfire-soc/polarfire-soc-documentation/blob/master/reference-designs-fpga-and-development-kits/updating-mpfs-kit.md#ubuntu-server-image

**Reply by rockoguido** [reply]
cool, I'm all over it, whatever results I get I'll post them here! 
thx!

**Reply by rockoguido** [reply]
ok got canonical deployed on my Icicle Kit, a nice start.   in my case I had an older bitstream in the fabric which allowed an older buildroot to boot but not canonical.    So, had to update the bitstream from https://github.com/polarfire-soc/icicle-kit-reference-design/releases
got the latest version 2022_10.
Now can boot Ubuntu and do whatever one would do if they just reimaged their host machine, lots of time spent with the package manager..  
For fun did some ethernet throughput benchmarking with Iperf, got 640 Mbits/sec,  respectable without doing any tuning, just default parameters.  
At this point building a new Buildroot image so that I can boot the Icicle Kit using the latest bitstream
Thx!

**Reply by rockoguido** [reply]
now with the 2023.02 bitstream for the Icicle Kit I can boot Buildroot, Ubuntu and Yocto 2023.02.     I'm feeling it! :)
